tests/publiccloud/nvidia.pm

```diff
@@ -17,8 +17,7 @@ sub validate_nvidia {
     validate_script_output("hwinfo --gfxcard", sub { /nVidia.*Tesla T4/mg });    # depends on terraform setup
         # nvidia-smi is delivered by nvidia-compute-utils-G06. Not available on PublicCloud
         # Check device files and loaded modules instead.
-    validate_script_output("lsmod", sub { m/nvidia/ }, fail_message => "nvidia module not loaded");
-    assert_script_run("ls -la /dev/{nvidia-modeset,nvidia-uvm-tools,nvidiactl,nvidia-uvm,nvidia0}", fail_message => "nvidia device files are missing");
+    validate_script_output("lsmod | grep nvidia", sub { m/nvidia|nvidia_modeset/ }, fail_message => "nvidia module not loaded");
```

Comment:
Are the nvidia modules loaded by defautl? Or should the test try to run `modprobe nvidia` before checking loaded modules?

Also I think it is not necessary to grep and then validate the filtered output. Feels bit overkill to me

Reply:
It comes loaded by default in the PC instances.